internal/broker/bind_create_test.go

```diff
@@ -376,11 +378,33 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		binding = unmarshal(t, response)
 		assert.Equal(t, firstInstanceSecondBinding, binding)
 		assert.Equal(t, firstInstanceSecondBindingDB.Kubeconfig, binding.Credentials.(map[string]interface{})["kubeconfig"])
-		assertClusterAccess(t, response, "secret-to-check-first", binding)
+		assertClusterAccess(t, "secret-to-check-first", binding)
+	})
+
+	t.Run("should delete created binding", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		createdBindingID, createdBinding := createBindingForInstance(instanceID, httpServer, t)
```

Comment:
Consider creation of more bindings to check if we delete selectively.
Having delete implemented by `DELETE * from bindings` we would pass this test.

Reply:
Why don't we pass `instanceID` and `bindingID` to `createBindingForInstance`?
Instead of creating random bindingID in the function?
We would have more control (using constant or creating random value on the caller level), it would be closer to real usage where the API caller passes both IDs.

Reply:
I am trying to chose the lesser of two evils. I am passing only instanceID since the value is predefined. All tests of bind_create_test.go file rely on two instances that need secrets with Kubeconfig to be created in the current K8S context. I do not want to do this setup in every test case to make the test faster. 

In case of bindingID I do not want to pass it as parameter because `createBindingForInstance` has already too many parameters. Additionally, generating the values make test repeatable because every time we relay on unique values that are not share between the test cases.

Reply:
Disagree - three parameters are not too many.